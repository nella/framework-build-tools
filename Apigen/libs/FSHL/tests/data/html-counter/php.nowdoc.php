<?php $text = <<<'TEXT'
hello world $say $something \$boo \\ {$foo->bar[1]}
TEXT;
?>