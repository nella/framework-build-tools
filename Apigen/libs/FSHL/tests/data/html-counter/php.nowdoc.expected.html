<html><head><meta http-equiv="content-type" content="text/html; charset=utf-8" /><link rel="stylesheet" type="text/css" href="../../../style.css" media="all" /></head><body><pre>
<span class="line">1: </span><span class="xlang">&lt;?php</span> <span class="php-var">$text</span> = <span class="php-quote">&lt;&lt;&lt;'TEXT'
</span><span class="line">2: </span><span class="php-quote">hello world $say $something \$boo \\ {$foo-&gt;bar[1]}
</span><span class="line">3: </span><span class="php-quote">TEXT;
</span><span class="line">4: </span><span class="xlang">?&gt;</span>
</pre></body></html>